soccer/src/soccer/planning/planner/motion_command.hpp

```diff
@@ -56,9 +56,19 @@ struct PathTargetCommand {
     bool operator==(const PathTargetCommand& ptc) {
         bool pos_eq = goal.position == ptc.goal.position;
         bool vel_eq = goal.velocity == ptc.goal.velocity;
-        // TODO(Kevin): fix this to actually compare std::variants
-        bool angle_eq = true;
-        /* bool angle_eq = goal.velocity == ptc.goal.velocity; */
+        bool angle_eq = false;
+        if (std::holds_alternative<TargetFaceTangent>(angle_override) &&
+            std::holds_alternative<TargetFaceTangent>(ptc.angle_override)) {
+            angle_eq = &(angle_override) == &(ptc.angle_override);
```

Comment:
small nit, but usually the & won't have parens around the target variable
```suggestion
            angle_eq = (&angle_override == &ptc.angle_override);
```